libraries-2/src/test/java/com/baeldung/jcommander/usagebilling/cli/UsageBasedBillingIntegrationTest.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.jcommander.usagebilling.cli;
+
+import org.junit.Test;
+
+public class UsageBasedBillingIntegrationTest {
+
+    @Test
+    public void whenInvokeSubmitCommand_thenProcessUsage() {
+        String[] argv = {
+          "submit",
+          "--customer", "cb898e7a-f2a0-46d2-9a09-531f1cee1839",
+          "--subscription", "subscriptionPQRMN001",
+          "--pricing-type", "PRE_RATED",
+          "--timestamp", "2019-10-03T10:58:00",
+          "--quantity", "7",
+          "--price", "24.56"
+        };
+        new UsageBasedBilling().run(argv);
```

Comment:
These tests have no assertions in them... I don't know if that's important.

Reply:
This was more about invoking the commands with the full set of inputs. I've removed the test and instead, added instructions in the main class.